app/controllers/settings/profiles_controller.rb

```diff
@@ -5,6 +5,10 @@ def show
   def update
     user_params_with_family = user_params
 
+    if params[:user][:profile_image].blank?
+      Current.user.profile_image.purge
+    end
```

Comment:
Since this form is tied to the update for a user's first and last name, the profile picture will be cleared out on every save to the form.

Repro:

1. Add picture
2. Hit save
3. Hit save again

Reply:
Oh good spot.